postgres-matrix.yml

```
@@ -0,0 +1,13 @@
+name: Postgres Version Matrix
+project_name: citus
+
+version_matrix:
+  - 8.0:
+      postgres_versions: [10, 11]
+  - 9.0:
+      postgres_versions: [11, 12]
+  - 9.5:
+      postgres_versions: [11, 12, 13]
+  # If 10.0 is released, since it is between 10.1 and 9.5, 9.5 support will be effective for 10.0.x releases
```

I think it makes more sense to have an entry for 10.0, instead of having a comment that is actually longer than the alternative.

```suggestion
  - 10.0:
      postgres_versions: [11, 12, 13]
```

---

I added it just to be easily understood no need to add an entry

---

ok. your call